SonarQube.TeamBuild.PreProcessor/ArgumentProcessor.cs

```diff
@@ -19,6 +20,12 @@ namespace SonarQube.TeamBuild.PreProcessor
     /// </summary>
     public static class ArgumentProcessor // was internal
     {
+        /// <summary>
+        /// Regular expression to validate a project key.
+        /// See http://docs.sonarqube.org/display/SONAR/Project+Administration#ProjectAdministration-AddingaProject
+        /// </summary>
+        private static readonly Regex ProjectKeyRegEx = new Regex(@"^[0-9a-zA-Z:-_\.]+$", RegexOptions.Compiled | RegexOptions.Singleline);
+
```

Comment:
@duncanpMS It's a bug in the documentation.

You'll find the exact regular expression used here: https://github.com/SonarSource/sonarqube/blob/5.1.1/sonar-core/src/main/java/org/sonar/core/component/ComponentKeys.java#L36

Please use `a-zA-Z` for alpha and `a-zA-Z0-9` for alphanum